### DISCUSSION TITLE
Emoji Fix - Feedback and Suggestions

### DISCUSSION THREAD

**Discussion by hgmich** [holmesmr/Blaseball-Userstyles#11] [Ideas]
If you have any suggestions, improvements or feedback for the Emoji Fix, please discuss them here!

Please don't post bug reports here! Please [file an issue](holmesmr/Blaseball-Userstyles/issues/new) if you're having a problem with the Emoji Fix.

**Comment by KarolisLive** [comment]
Hi!
There's a new place where emojis show up -- next to pitchers and batters (ref. https://twitter.com/SIBROfficial/status/1366859873661583362 ),
Looks like those places are not "emoji fixed" yet.

**Reply by hgmich** [reply]
Thanks - a fix for this should be out soon.

**Comment by ashktree** [comment]
Hi, when I use the emoji fix, it turns text at various points on the website black instead of white. I've most noticed this in the election feed and the player feed. screenshots below. thanks!

w/o emoji fix: 
![image](https://user-images.githubusercontent.com/53458334/113515581-5668fa80-9543-11eb-8b49-f75a36453075.png)
w/ emoji fix:
![image](https://user-images.githubusercontent.com/53458334/113515538-173aa980-9543-11eb-96bc-d468375d377e.png) 

w/o emoji fix: 
![image](https://user-images.githubusercontent.com/53458334/113515588-62ed5300-9543-11eb-995b-3b1215560fc3.png)
w/ emoji fix:
![image](https://user-images.githubusercontent.com/53458334/113515560-4224fd80-9543-11eb-97b7-570df31015c8.png)

**Comment by goblinzone** [comment]
Heya, seems the issues with black text are happening because the style's looking for a .dark-mode and .light-mode class that aren't present in the site code rn. Was able to fix it locally by removing the black colour applied by the big first font change. I'll put in a pull request shortly.
